feature/bgp/gracefulrestart/1

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Hmm, did you copy this file in error instead of updating the gracefulrestart.go file?

Reply:
Removed. Done!